### PR TITLE
Cockpit 267

### PR DESCRIPTION
**Pull request by garrett**
Preview: https://garrett.github.io/cockpit-project.github.io/blog/cockpit-267.html

### CODE REVIEW

**Comment by martinpitt** [comment]
@garrett : Thanks! I updated the "merged" PRs a bit. Can you please regenerate and do a round of verbiage refinement today already? I don't have much time tomorrow, and would like to prepare the release as much as possible today already.

**Comment by garrett** [comment]
I regenerated it with the updates.

Checking the content will have to wait until tomorrow; I feel pretty bad right now and hopefully will be better in the morning.

**Comment by garrett** [comment]
Oddly, GitHub isn't updating it. The date should say April 12. It's in Git though.

**Comment by garrett** [comment]
Oh. jQuery was removed in that big PR. It shouldn't be. The site needs it. It won't compile without it, and search uses it. We need to add it back, else the site won't work.

**Comment by martinpitt** [comment]
@allisonkarlitskaya : Can you please have a first round of English beautification? (only for the "merged" bits; I don't expect us to land the open stuff by tomorrow morning)

File: _posts/2022-04-12-cockpit-267.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+Redirection of a channel is attempted on all bridges - the shortest path then takes precedence (e.g. if there is a redirect between two channels on the same peer, it happens on both the parent bridge and the peer, but effectively the data is redirected on the peer and nothing arrives on the parent). This also means it is compatible with older versions of cockpit-bridge - they simply ignore the redirect option and the redirect is done on the parent bridge (this can obviously be slower, but it will work).
+
+I hope this makes sense - it seems like a small, but useful feature to me. (For potential uses see the linked issue.)
+
+## Machines: Remove static hosts from virtual network's DHCP
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit-machines/pull/640>
+{:.note}
+
+The "static host entries" now have a "remove" action link:
```
Comment (allisonkarlitskaya) [inline _posts/2022-04-12-cockpit-267.md:67]: "action link" sounds very web developer lingo-y :)

File: _posts/2022-04-12-cockpit-267.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+![network_overview]({{ site.baseurl }}/images/267-machines-remove-static-hosts-from-virtual-network-s-dhcp.png)
+
+## Machines: Redesign VM creation
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit-machines/pull/560>
+{:.note}
+
+The content of the Virtual machine creation dialog is split between a mandatory Details configuration and an optional Automation configuration.
```
Comment (allisonkarlitskaya) [inline _posts/2022-04-12-cockpit-267.md:76]: s/is split between/has been split into/

**Review by allisonkarlitskaya — CHANGES_REQUESTED** [review]

File: _posts/2022-04-12-cockpit-267.md (hunk 1 above)
Comment (martinpitt) [reply _posts/2022-04-12-cockpit-267.md:67]: Good point -- changed to just "action" in https://github.com/cockpit-project/cockpit-machines/pull/640 (will pick up on next refresh, but that needs #515 to land)

**Comment by martinpitt** [comment]
Rebased to the jquery drop revert, and refreshed for release mode.

File: _posts/2022-04-13-cockpit-267.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+The crypto policy dialog can now enable or disable FIPS mode, using the [fips-mode-setup command](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/assembly_installing-a-rhel-8-system-with-fips-mode-enabled_security-hardening). It will also validate that the kernel and user-space configuration are consistent, and allow you to fix them when necessary.
+
+![screenshot of fips crypto policy support]({{ site.baseurl }}/images/267-fips-crypto-policy-support.png)
+
+## Machines: Redesign VM creation
+
+The content of the Virtual machine creation dialog has been split into a mandatory Details configuration and an optional Automation configuration.
+
+Clicking on "Create and run" will create, immediately start, and install the VM. Clicking on "Create and edit" will create the VM in a shut-off state and redirect the user to the details page, where further configuration of the VM may be done before the installation is started.
```
Comment (garrett) [inline _posts/2022-04-13-cockpit-267.md:29]:
```suggestion
Clicking on "Create and run" will create, immediately start, and install the VM. Clicking on "Create and edit" will create the VM in a "shut-off" state and redirect the user to the details page, where the VM may be further customized before starting installation.
```

The sentence is rather wordy, so I dropped a few words and reorganized it a little.

File: _posts/2022-04-13-cockpit-267.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+![details_tab]({{ site.baseurl }}/images/267-machines-redesign-vm-creation.png)
+
+![automation_tab]({{ site.baseurl }}/images/267-machines-redesign-vm-creation-2.png)
+
+## Machines: Remove static hosts from virtual network's DHCP
+
+The "static host" entries now have a "remove" action:
```
Comment (garrett) [inline _posts/2022-04-13-cockpit-267.md:37]:
```suggestion
The "static host" entries now have a "remove" action.
```

We probably don't need a colon here to introduce the image.

**Review by garrett — COMMENTED** [review]
I've made some suggestions.

**Review by allisonkarlitskaya — APPROVED** [review]
Reads fine to me.

**Comment by martinpitt** [comment]
Added bodhi URLs.

**Pull request merged**